scripts/release/create_binarydeb_task_generator.py

```diff
@@ -100,6 +102,8 @@ def main(argv=sys.argv[1:]):
         'dependencies': debian_pkg_names,
         'dependency_versions': debian_pkg_versions,
 
+        'build_environment_variables': args.env_vars,
```

Comment:
Nitpick: move this line three up so that it is passed in the order it is being used in the template.